src/chia_log/log_consumer.py

```diff
@@ -50,53 +54,69 @@ def _notify_subscribers(self, logs: str):
 
 
 class FileLogConsumer(LogConsumer):
-    """Specific implementation for a simple file consumer"""
-
     def __init__(self, log_path: Path):
-        logging.info("Enabled file log consumer.")
         super().__init__()
         self._log_path = log_path
+        self._expanded_log_path = ""
         self._is_running = True
         self._thread = Thread(target=self._consume_loop)
         self._thread.start()
+        self._log_size = 0
 
     def stop(self):
         logging.info("Stopping")
         self._is_running = False
 
+    @retry((FileNotFoundError, PermissionError), delay=2)
```

Comment:
Does this silently retry or it'll print something out? If the user specifies wrong path or has incorrect permissions setup, it should alert them in some way.

Maybe a good idea to set a limit on number of retries, e.g. `tries=3` 🤔

Reply:
This prints out the following in the log
`[2021-05-21 23:12:45] [ WARNING] --- [WinError 2] The system cannot find the file specified: 'C:\\Users\\PieterHelsen\\workspace\\debug.log', retrying in 2 seconds... (api.py:40)`

As per retries, that was how I implemented it first, but since the retry count is on the _consume_loop, this makes it stop working after 3 bad rotations. The retry counter doesn't seem to get reset.